integration/playwright.config.ts

```diff
@@ -26,8 +26,13 @@ const config: PlaywrightTestConfig = {
       use: devices["Desktop Safari"],
     },
     {
-      name: "edge",
-      use: devices["Desktop Edge"],
+      name: "msedge",
+      use: {
+        ...devices["Desktop Edge"],
+        // Desktop Edge uses chromium by default
+        // https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/deviceDescriptorsSource.json#L1502
+        channel: "msedge",
```

Comment:
oops - we weren't actually using edge before 🙈 

https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/deviceDescriptorsSource.json#L1502